src/codeu/chat/common/ServerInfo.java

```diff
@@ -0,0 +1,29 @@
+package codeu.chat.common;
+import codeu.chat.util.Uuid;
+import java.io.IOException;
+
+public final class ServerInfo {
+
+  private final static String SERVER_VERSION = "1.0.0";
+
+  public final Uuid version;
+
+  //sets the server version to either null or
+  public ServerInfo() {
+    Uuid temp = null;
+    try{
+      temp =  Uuid.parse(SERVER_VERSION);
+    } catch(IOException e){
+      System.out.println("Server could not parse the user id");
```

Comment:
The tutorial for the technical activities provided that code, I just added the try catch 
![image](https://cloud.githubusercontent.com/assets/21254501/26658051/91d2a7b2-461c-11e7-801d-e0b59355d590.png)